src/javascript/binary_japan/trade_japan/JPTradePage.js

```diff
@@ -0,0 +1,69 @@
+var JPTradePage = (function() {
+
+  var isJapan = true;
+  var documentReady = false;
+
+  var onLoad = function() {
+    isJapan = true;
+    
+    $(function(){
+      JapanTrading.start();
+      documentReady = true;
+    });
+
+    if(documentReady){
+      JapanTrading.start();
+    }
+
+    Content.populate();
+    TradingAnalysis.bindAnalysisTabEvent();
+    $('#tab_portfolio a').text(text.localize('Portfolio'));
+    $('#tab_graph a').text(text.localize('Chart'));
+    $('#tab_explanation a').text(text.localize('Explanation'));
+    $('#tab_last_digit a').text(text.localize('Last Digit Stats'));
+    $('#tab_japan_info a').text(text.localize('Prices'));
+
+    window.chartAllowed = true;
+    handleChart();
+  };
+
+  var handleChart = function() {
```

Comment:
@binary-maksym if you're not changing the handleChart function, I think you can just call it on line 27 with `TradePage.handleChart()` instead of copying the function entirely again.